packages/media-utils/src/components/media-upload/index.js

```diff
@@ -26,21 +31,42 @@ const getFeaturedImageMediaFrame = () => {
 			} );
 		},
 
+		/**
+		 * Handle the edit state requirements of selected media item.
+		 *
+		 * @return {void}
+		 */
+		editState( ) {
+			const selection = this.state( 'featured-image' ).get( 'selection' ),
```

Comment:
Nitpick, but I'd prefer separate `const` declarations for each variable rather than the shorthand because it makes it easier to read at a glance. I'd also move `this.content.set(view);` to a new line to separate functional code from the variable declarations and add an inline comment.

Reply:
Yes, variable declarations are usually in their own statements in Gutenberg. I'm familiar with both approaches because of previous experience, but I agree that we could align.